dailiza-bot/text_patterns.py

```diff
@@ -35,7 +35,7 @@
     [r"[Ww]ie [Hh]ei[sß]s*t du?",
```

Comment:
the pattern doesn't have to include punctuations (?/./,/!) because the user_input has already been stripped of them in the dailiza_bot.py file

Reply:
Thanks i will have to change it